File: flutter_stock_trading_app/lib/models/stock.dart
class Stock {
  final int stockId;
  final String stockName;
  final double currentPrice;

  const Stock({
    required this.stockId,
    required this.stockName,
    required this.currentPrice,
  });

  factory Stock.fromJson(Map<String, dynamic> json) {
    return Stock(
      stockId: json['stockID'],
      stockName: json['stockName'],
      currentPrice: json['currentPrice'],
    );
  }

  Map<String, dynamic> toJson() => {
        'stockId': stockId,
        'stockName': stockName,
        'currentPrice': currentPrice,
      };
}

File: Backend_StockTradingApp/StockTradingApp/Models/Stock.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace StockTradingApp.Models
{
    public class Stock
    {
        [Key]
        public int StockID { get; set; }
        public string StockName { get; set; }
        public decimal CurrentPrice { get; set; }
    }
}